docker-compose.yml

```diff
@@ -27,6 +27,9 @@ services:
       - pg_data:/var/lib/postgresql/data/:cached
     restart: on-failure
     container_name: "data-submission-service-api_db"
-
+  redis:
```

Comment:
Do you also need to add this to the `depends_on` section above?

Reply:
good shout, updated!